go/adbc/pkg/Makefile

```diff
@@ -25,20 +29,22 @@ else
 	SUFFIX=dylib
 endif
 
-DRIVERS := \
-	libadbc_driver_bigquery.$(SUFFIX) \
-	libadbc_driver_databricks.$(SUFFIX) \
-	libadbc_driver_flightsql.$(SUFFIX) \
-	libadbc_driver_panicdummy.$(SUFFIX) \
-	libadbc_driver_snowflake.$(SUFFIX)
+# Expand dynamically libadbc_driver_.SUFFIX
+DRIVERS := $(addsuffix .$(SUFFIX),$(addprefix libadbc_driver_,$(MANAGERS)))
+
+# =========
+#  Targets
+# =========
 
 .PHONY: all regenerate
 all: $(DRIVERS)
 
-libadbc_driver_%.$(SUFFIX): %
-	$(GO_BUILD) -tags driverlib -o $@ -buildmode=c-shared -ldflags "-s -w" ./$<
+
+libadbc_driver_%.$(SUFFIX): % ../driver/% ../go.mod ../go.sum
```

Comment:
as we add more files, we can add more dependencies to track on the RHS of this line